src/condor_utils/condor_scitokens.cpp

```diff
@@ -175,6 +180,24 @@ htcondor::validate_scitoken(const std::string &scitoken_str, std::string &issuer
 			(*enforcer_acl_free_ptr)(acls);
 		}
 
+		char *jti_char = nullptr;
+		if (!(*scitoken_get_claim_string_ptr)(token, "jti", &jti_char, nullptr)) {
+			if (jti_char) {jti = jti_char;}
+			free(jti_char);
+		}
+
+		char **group_list;
+		if (scitoken_get_claim_string_list_ptr &&
+			!(*scitoken_get_claim_string_list_ptr)(token, "wlcg.groups", &group_list, nullptr) &&
+			group_list)
+		{
+			int idx = 0;
+			char *grp;
+			while ( (grp = group_list[idx++]) ) {
+				groups.emplace_back(grp);
+			}
+		}
```

Comment:
Are you missing a `!(*scitoken_free_string_list_ptr(group_list);` here?

Reply:
yes, good catch!